### ISSUE TITLE
Odd cycle-focus screen behavior after layout is applied

### ISSUE THREAD

**Issue by chrishoage** [koekeishiya/kwm#233]
After applying a layout in an multi-display environment with `cycle-focus screen` set, some windows are ignored and focus is cycled across displays (as if `cycle-focus all` is set). 

This behavior persists until the 2nd display is focused. Once this display is focused and focus is given to display 1 again, `cycle-focus screen` behaves as expected.

**Comment by koekeishiya** [comment]
> cycle-focus all

this has been removed, didn't really see a reason to keep it around.

> This behavior persists until the 2nd display is focused. Once this display is focused and focus is given to display 1 again, cycle-focus screen behaves as expected.

I am able to reproduce this. This appears to be happening if Kwm has not yet initialized the secondary monitor. Once the secondary monitor is focused once, and the tree is created, things starts working as expected. No idea yet as to why this is, but probably something I overlooked when rewriting the tree structure.
